wazo_confd/plugins/email/schema.py

```diff
@@ -1,54 +1,53 @@
 # Copyright 2021 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from marshmallow import fields, post_load, pre_dump
+from marshmallow import fields, post_load, pre_dump, ValidationError
 from marshmallow.validate import Length
 
 from wazo_confd.helpers.mallow import BaseSchema
 
 
 class _RewriteRule(BaseSchema):
-    match = fields.String()
-    replacement = fields.String()
+    match = fields.String(required=True)
+    replacement = fields.String(required=True)
+
+    @pre_dump
+    def from_canonical(self, data, **kwargs):
+        if data and ' ' not in data:
+            raise ValidationError(
+                f'Data value "{data}" should be of format "match replacement"'
+            )
```

Comment:
dump never raise ValidationError
You can leave it fail of raise a standard exception (500), because the user cannot do anything to fix it (the error is in the database)

Reply:
Indeed, fixed